src/studio/src/designer/frontend/packages/schema-editor/src/components/SchemaInspector.tsx

```diff
@@ -464,26 +514,26 @@ const SchemaInspector = ((props: ISchemaInspectorProps) => {
                   checked={isRequired} onChange={handleRequiredChanged}
                   name='checkedRequired'
                 />}
-                label={getTranslation('schema_editor.required', props.language)}
+                label={getTranslation('required', props.language)}
               />
             </Grid>
             <Grid item xs={12}>
               <hr className={classes.divider} />
             </Grid>
             <Grid item xs={4}>
-              <p>{getTranslation('schema_editor.keyword', props.language)}</p>
+              <p>{getTranslation('keyword', props.language)}</p>
             </Grid>
             <Grid item xs={1} />
             <Grid item xs={7}>
-              <p>{getTranslation('schema_editor.value', props.language)}</p>
+              <p>{getTranslation('value', props.language)}</p>
             </Grid>
             { itemToDisplay && renderItemRestrictions(itemToDisplay) }
           </Grid>
           <IconButton
             id='add-restriction-button'
-            aria-label={getTranslation('schema_editor.add_restriction', props.language)}
+            aria-label={getTranslation('add_restriction', props.language)}
             onClick={onAddRestrictionClick}
-          ><i className='fa fa-plus'/>{getTranslation('schema_editor.add_restriction', props.language)}
+          ><i className='fa fa-plus'/>{getTranslation('add_restriction', props.language)}
```

Comment:
Why do we use these self-breaking <i /> tags?
Seems a bit hacky...

Reply:
standard fontawesome way of rendering icons